src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -313,94 +313,133 @@ protected static void DisplayManifestPreview(Manifests manifests)
         }
 
         /// <summary>
-        /// Validates the GitHubToken provided on the command-line, or if not present, the cached token if one exists.
-        /// Attempts a simple operation against the target repo, and if that fails, then:
-        ///     If token provided on command-line, errors out
-        ///     If not, and cached token was present, then deletes token cache, and starts OAuth flow
-        /// Otherwise, sets the instance variable to hold the validated token.
-        /// If no token is present on command-line or in cache, starts the OAuth flow to retrieve one.
+        /// Creates a new GitHub client using the provided or cached token if present.
+        /// If the requireToken bool is set to TRUE, then the OAuth flow is launched to acquire a new token for the client.
```

Comment:
Clarify that Oauth is only launched if there's no token provided in command line or in cache

Reply:
Added additional clarification message